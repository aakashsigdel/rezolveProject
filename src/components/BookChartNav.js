import React from 'react'
import { Link } from 'react-router'

import './BookChartNav.scss'

const BookChartNav = props =>
  <div className="nav-container">
    <Link to="/" activeClassName="nav-link-active" className="nav-link" onlyActiveOnIndex={true}>
      MY BOOKS
    </Link>
    <div className="nav-element-separator">&nbsp; | &nbsp;</div>
    <Link to="/charts" activeClassName="nav-link-active" className="nav-link">
      CHARTS
    </Link>
  </div>

export default BookChartNav
