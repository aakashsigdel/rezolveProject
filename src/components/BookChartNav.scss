@import '../variables';

.nav-container {
  display: flex;
  margin-top: 20px;
}

.nav-link {
  font-size: 18px;
  color: $nav_font;
}

.nav-link-active {
  font-size: 18px;
  font-weight: bold;
  color: $active;
}

.nav-element-separator {
  font-size: 18px;
  font-weight: bold;
  color: $nav_font;
}
