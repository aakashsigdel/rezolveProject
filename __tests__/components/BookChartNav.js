import React from 'react'
import BookChartNav from '../../src/components/BookChartNav'

import renderer from 'react-test-renderer'

it('should render <BookChartNav /> correctly', () => {
  const tree = renderer.create(
    <BookChartNav />
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
