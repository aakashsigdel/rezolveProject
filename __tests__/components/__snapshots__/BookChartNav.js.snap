exports[`test should render <BookChartNav /> correctly 1`] = `
<div
  className="nav-container">
  <a
    className="nav-link"
    onClick={[Function]}
    style={Object {}}>
    MY BOOKS
  </a>
  <div
    className="nav-element-separator">
      |  
  </div>
  <a
    className="nav-link"
    onClick={[Function]}
    style={Object {}}>
    CHARTS
  </a>
</div>
`;
